filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -9,44 +9,28 @@
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * WITHOUT WARANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
`WARRANTIES` is correct

Reply:
oh that's weird. I don't know how it got changed. good catch.